tests/regression/instance_segmentation/test_instance_segmentation.py

```diff
@@ -172,7 +172,7 @@ def test_otx_train_cls_incr_kpi_test(self, template):
 
     @e2e_pytest_component
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
-    @pytest.mark.skip(reason="Issue#2059: Two stage detector shows 0.0 mAP when OpenVINO exported in torch 1.13")
+    @pytest.mark.skip(reason="Issue#2290: MaskRCNN shows degraded performance when inferencing in OpenVINO")
```

Comment:
As far as I know, exporting model trained on large dataset doesn't show performance drop. Please correct me if I'm wrong

Reply:
It's contradictive. Sometimes a minor drop happens, like with action detection